src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+:root {
+  --blue: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  color: black;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
```

Comment:
```suggestion
```

Reply:
Thank you! Made all the changes. Ready for checking